src/main/resources/org/csanchez/jenkins/plugins/kubernetes/PodTemplate/config.jelly

```diff
@@ -17,6 +17,14 @@
     <f:textbox/>
   </f:entry>
 
+  <f:entry title="${%Usage}" help="/help/system-config/master-slave/usage.html">
```

Comment:
I tried to maintain the old behaviour with this change which was using the Node.Mode.NORMAL for the dynamic nodes. Would you rather have it set to Node.Mode.EXCLUSIVE?